bitnami/redis/Chart.yaml

```diff
@@ -34,4 +34,4 @@ maintainers:
 name: redis
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 18.11.0
+version: 18.12.0
```

Comment:
Would this not need to be `18.13.0`

Looks to be failing the tests due to non-matching version bumps `num_version_bumps`

```
script: let message='Detected changes in charts without version bump in Chart.yaml. Charts changed: 2. Version bumps detected: 1';
```

Reply:
As this is not adding any feature, I'd say it should be a patch bump